src/components/layout/Footer.vue

```diff
@@ -3,12 +3,19 @@
     v-spacer
       | © {{ new Date().getFullYear() }} - Made with 
       b.red--text ❤ 
-      | by QA
+      | by 
+      a(href="https://www.wolox.com.ar/")
+        | WOLOX
 </template>
 
 <style lang="scss" scoped>
 .footer {
-  background: none;
+  background-color: #363636;
```

Comment:
Add all the colors to a separated file named `_colors.scss` on a `variables` folder and import it to every component you need it